### DISCUSSION TITLE
Huawei Watch GT2 as a beacon?

### DISCUSSION THREAD

**Discussion by nashaden** [ESPresense/ESPresense#476] [Q&A]
Can anyone confirm or deny Huawei Watch GT2 working as a beacon with ESPresense?

**Comment by gnome4711** [comment]
My experience:
The Huawei GT2 only advertises when it is not connected to a smartphone. So it seems the watch advertises only to be found. After connection to my smartphone the GT2 doesn't show up anymore.

**Reply by DTTerastar** [reply]
Added it to the known not to work, thanks @gnome4711!
